apps/explorer/lib/explorer/chain.ex

```
@@ -706,12 +711,18 @@ defmodule Explorer.Chain do
           | {:error, step :: Ecto.Multi.name(), failed_value :: any(),
              changes_so_far :: %{optional(Ecto.Multi.name()) => any()}}
   def import_blocks(options) when is_list(options) do
+    broadcast =
+      case Keyword.fetch(options, :broadcast) do
```

Is the key `:broadcast` optional or it always must be in the `options` list? If it is required, I think we can go with `fetch!` and let it crash when for some reason the key was not given.

---

It is optional.